app/views/distributions/leap-15.1.html.erb

```diff
@@ -0,0 +1,155 @@
+<header class="download-home-header jumbotron jumbotron-fluid text-center">
+  <div class="container">
+    <div class="page-heading">
+      <%= leap_image_tag alt: "Leap 15 Logo", class: "animated jackInTheBox", width: "256", height: "256" %>
+      <h1 class="display-3">
+        openSUSE Leap <%= @version %><% if @testing_version.present? %> <sup><%= @testing_state %></sup><% end %>
+      </h1>
+    </div>
+  </div>
+</header>
+
+<section class="my-5">
+  <div class="container">
+    <!-- Nav tabs -->
+    <ul class="nav nav-tabs nav-justified" role="tablist">
+      <li class="nav-item">
+        <a class="nav-link active" data-toggle="tab" href="#official-ports" role="tab">
+          <%= _("Installation") %>
+          <span class="badge badge-success">x86_64</span>
+        </a>
+      </li>
+      <% if @have_live_images.present? %>
```

Comment:
Just for clarification: Is this a new instance variable that we will add to the controller later on?

Reply:
maybe. I tried a commented block first but that didn't work properly so I used this way. If you have a better idea how to disable that part of the template without completely removing let me know :)

Reply:
We could turn the `@have_live_images` into a helper function that takes the (leap) version as argument: `if has_live_images('15.1')`  or (for now) have a version specific variable (`@live_images_15_1.present?`). That way it will be easier to manage that part later on.

We will also have to think about how this will work on the inside (and how the information is provided), but that can be done later on.

Reply:
whatever. if this leads to more work I'll just delete block from the template and insert  it later when we have lives again.

Reply:
May be we can tweak https://github.com/openSUSE/software-o-o/blob/master/config/releases.yml to put the array into a key "state" or "history", and have another section with metadata for each release that we can use in templates.

However, I am uncomfortable that this is becoming yet another data source. There is already a `/distributions` call to the API that should gather each release metadata but everything going into this app because nobody wants to put things in OBS.